### ISSUE TITLE
route /  not work  on   php built-in server 

### ISSUE THREAD

**Issue by tablecell** [janniks/Restable#2]
php -S localhost:80

visist http://localhost 

browser display  error:"404 - not found"

**Comment by tablecell** [comment]
if url is  'http://localhost' 
self::request_path is empty string ""  but  register route is  '/'

**Comment by janniks** [comment]
Good catch! Unfortunately, it's been a while since I touched PHP — so, this likely will not be fixed...
